docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -290,6 +265,15 @@ Release
 
    Wait for the build to complete successfully.
 
+#. Set up ``AWS Access Key ID`` and ``AWS Secret Access Key`` Amazon S3 credentials:
```

Comment:
This is done in the Vagrant box which is created at the start of each release process. Also, it is done once per person already sort-of: the next line explains that the `~/.aws` configuration is copied from the host if it exists, and the end of the release process copies it from the VM to the host.